packages/central-server/src/apiV2/import/importSurveys/importSurveyQuestions.js

```diff
@@ -103,14 +109,19 @@ export async function importSurveysQuestions({ models, file, survey, dataGroup,
   let currentScreen;
   let currentSurveyScreenComponent;
   let hasSeenDateOfDataQuestion = false;
+  const errors = []; // An array to hold all errors, allowing multiple errors to be thrown at once
   const questionCodes = []; // An array to hold all question codes, allowing duplicate checking
   const configImporter = new ConfigImporter(models, rows);
   const objectValidator = new ObjectValidator(constructQuestionValidators(models));
   let hasPrimaryEntityQuestion = false;
   for (let rowIndex = 0; rowIndex < rows.length; rowIndex++) {
     const row = rows[rowIndex];
-    const constructImportValidationError = (message, field) =>
-      new ImportValidationError(message, excelRowNumber, field);
+    const constructImportValidationError = (message, field, { details }) => {
+      return {
+        message: new ImportValidationError(message, excelRowNumber, field).message,
+        details,
+      };
+    };
```

Comment:
```suggestion
    const constructImportValidationError = (message, field, { details }) =>
      new ImportValidationError(message, excelRowNumber, field, undefined, details);
```
I think we still want this function to be returning an error, as when we pass it into `ObjectValidator` that's what it's expected to do. We can convert the errors into the wrapped objects latter when we create the `MultiValidationError`